packages/block-library/src/legacy-widget/edit/preview.js

```diff
@@ -6,13 +6,44 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import { useRefEffect } from '@wordpress/compose';
 import { addQueryArgs } from '@wordpress/url';
 import { useState } from '@wordpress/element';
 import { Placeholder, Spinner, Disabled } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 export default function Preview( { idBase, instance, isVisible } ) {
-	const [ iframeHeight, setIframeHeight ] = useState( null );
+	const [ iframeHeight, setIframeHeight ] = useState();
```

Comment:
Why removing `null`? I believe it was served to be the default initial value as the loading state.

Maybe we should just use a loading state here, and set the height directly via DOM API (`iframe.style.height`).

Reply:
Ah, ok, didn't notice that. I think using `null` in that way is definitely confusing. I appreciate the change in #33191, which improves that.